website/s3_logger.py

```diff
@@ -1,38 +1,70 @@
 import logging
+import json
+import os
 
 from . import aws_helpers, log_queue
 
 logger = logging.getLogger(__name__)
 
 
+should_log_activity = os.getenv("LOG_USER_ACTIVITY")
```

Comment:
If you're making `s3_logger` generic, then this should probably go into `user_activity.py`, because it deals with that.